### ISSUE TITLE
Improve down-vote handling

### ISSUE THREAD

**Issue by s1037989** [KeystoneIT/Ama#161]
From Keith: When a question drops to 0 points, it is completely disappearing.  I accidentally clicked on the downvote button, and the question that was previously up there (that I'm sure was a very important question) is no longer visible.

**Comment by s1037989** [comment]
It appears that you're dropping a question once vote level is 0.  It should be lower, like -2 or so.

**Comment by s1037989** [comment]
Also, Julie informed me that a question was no longer showing up (the one mentioned in this issue) and so I searched the database for it.  It appears that it was deleted.  The question should _not_ be deleted, just simply not shown.  Or, use the filter method and default to not showing under -2, but allow showing all regardless of vote level.  I like that.  Do it.

**Comment by jsiems** [comment]
Vote floor is set in the .conf file:
e.x. vote_floor => -2
That should fix the issue of them being deleted when they are under 0, I think? Unless it is already set to -2 and they are being deleted anyway, that would not be good...

**Comment by s1037989** [comment]
Ah!  Of course!  I didn't update the prod config file.  Thanks!

**Comment by s1037989** [comment]
Yeah, and make sure that questions are _not_ deleted by downvotes.

Also, update the code to disable this "algorithm" when vote_floor is not set.  That's a bad idea when a config isn't set and then questions get deleted!
